lib/cli-engine.js

```diff
@@ -256,6 +256,27 @@ function isErrorMessage(message) {
     return message.severity === 2;
 }
 
+
+/**
+ * Checks if a provided path is a directory and returns a glob string matching
+ * all files under that directory if so, the path itself otherwise.
+ * Also makes sure all path separators are POSIX style for `glob` compatibility.
+ *
+ * @param {string} pathname The directory path to be modified
+ * @returns {string} The glob path or the file path itself
+ */
+function processPath(pathname) {
```

Comment:
Why do we need to do this?
Is it because `globby` doesn't understand a a directory path directly?